tests/e2e/local.web.docker-compose.yml

```diff
@@ -39,6 +40,7 @@ services:
       context: ./../../
       dockerfile: Dockerfile
     volumes:
-      - ./plugins:/root/.redisinsight-preview/plugins
+#      - ./plugins:/root/.redisinsight-v2/plugins
```

Comment:
Is that ok?

Reply:
Removed from both web and local docker-compose files. Thanks!